configs/stage/roles/content-sources.json

```diff
@@ -2,7 +2,7 @@
     "roles": [
       {
         "name": "Repositories administrator",
-        "description": "Perform any available operation against any repositories resource.",
+        "description": "Perform any available operation on any repositories resource.",
```

Comment:
please for this change is needed to increase the version

Reply:
"version": 2,   is now "version": 3,